structured-event-log/src/main/java/org/apache/pulsar/structuredeventlog/slf4j/Slf4jEvent.java

```diff
@@ -36,6 +39,9 @@
 import org.slf4j.LoggerFactory;
 
 class Slf4jEvent implements Event {
+    private static final ThreadLocal<Map<Object, Logger>> loggersTLS = ThreadLocal.withInitial(() -> new HashMap<>());
+    private static final Logger stringLogger = LoggerFactory.getLogger("stevlog");
```

Comment:
what about `org.apache.pulsar.structured.event.log` ?
it is better that the logger name starts with "org.apache.pulsar", in order to let the users shutdown Pulsar stuff buy using only "org.apache.pulsar" prefix